packages/ketcher-core/src/application/render/renderStruct.ts

```diff
@@ -59,6 +59,9 @@ export class RenderStruct {
     el: HTMLElement | null,
     struct: Struct | null,
     options: any = {},
+    originalStruct: Struct,
+    update = false,
+    monomerConnectionMode = false,
```

Comment:
I would propose to rename it to something more common, like needRescale or somethingLike that